Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -6,13 +6,15 @@ import fs = require('fs');
 
 import {ToolRunner} from 'vsts-task-lib/toolrunner';
 
+import tl = require('vsts-task-lib/task');
```

Comment:
I don't think those three listed functions are suitably related enough to put in a class together (unless that class is called "common"). _getStagingDir()_ and _getRunSettings()_ have nothing to do with TFS.